test/index.spec.tsx

```diff
@@ -209,6 +210,27 @@ describe('react-inlinesvg', () => {
       );
     });
 
+    it('should not uniquify non-id hrefs', async () => {
+      const original = await setup({
```

Comment:
I don't think is necessary to create an `original` instance to compare data.
Just check if the wrapped component with `uniquifyIDs` set to `true` doesn't change the prop.

Reply:
How would I do that? With a snapshot?